cli/requirements.txt

```diff
@@ -2,4 +2,5 @@ sphinx>=1.3.1, <2.0
 tox>=2.2, <3.0
 wheel>=0.24.0, <1.0
 pytest>=2.9.1
+teamcity-messages>=1.20
```

Comment:
Yes, we only run integration tests for the binary CIs which are in the 'cli' directory. Okay, makes sense to me.